Kang-Minseokk/Invaders-SDP
# KAN-38 feat : Added Fever time sound Effect

## What
This PR adds the functionality to play the "fever time" sound effect. It includes the following changes:
1. Configured the "fever time" sound file to be played as an effect sound (ES) instead of background music (BGM).
2. Set the duration of the sound effect to 5 seconds.

## Why
1. The "fever time" sound effect is an important part of the user experience and needs to be played as a short effect sound instead of a background music track.

## Related Issue

## How to Test
1. Start the application locally.
2. Trigger the event that initiates the "fever time" sound effect.
3. Ensure that the sound effect is played for 5 seconds.
4. Verify that the sound is not played as background music (BGM) but as a sound effect (ES).

## Screenshots

## Additional Information
The sound effect will automatically stop after 5 seconds.
